troubleshooting/tools/invalid-utf-8/invalid.py

```diff
@@ -0,0 +1,33 @@
+# small script that writes byte values to a file
+# which if interpreted as utf-8 will be invalid chars
+# can test FLB processing of invalid unicode
+
+myfile = open('log.log', 'ab')
+
+# from https://github.com/fluent/fluent-bit/pull/4297
+invalid = b'\xEF\xBF\x00'
+
+# these were copied from invalid utf-8 examples online
+other_invalid_byte_sequence=b'\xC2\xC2\xC0\xC1'
+invalid_examples = [
+    b'\xc3\x28',
+    b'\xa0\xa1',
+    b'\xe2\x28\xa1',
+    b'\xe2\x82\x28',
+    b'\xf0\x28\x8c\xbc',
+    b'\xf0\x90\x28\xbc',
+    b'\xf0\x28\x8c\x28',
+]
```

Comment:
Why don't we put them into one array ?? Why need to one array and two single `string` ? so line 26 - line 27 can be dropped.